code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -105,10 +105,6 @@
 
 		update_muts=1
 
-	if(type == /mob/living/carbon/monkey)
```

Comment:
Don't change code without testing it kids.

First, this check is always returning false, because it doesn't have "" around the type (which is a text variable thanks to BYOND).

Second, if this WERE working, the following monkeys would be the ONLY monkeys to have a language: Monkeys, Diona, Rocks. The rest have no language both as-is and if the check were working.